packages/web/components/form/validation/primary.tsx

```diff
@@ -0,0 +1,14 @@
+import * as yup from 'yup';
+import { FieldsType } from '.';
```

Comment:
Is there any chance of circular module load problem. Because from index.tsx ... primary tsx is imported?

Reply:
I don't believe so, I've never experience circular dependency issues in Next/CRA